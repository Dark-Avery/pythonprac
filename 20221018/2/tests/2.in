:sin1 sin(1)
sin1
:decorate "--<<{}>>--".format("UraUra")
decorate
quit "все будет хорошо!! {}, {}"