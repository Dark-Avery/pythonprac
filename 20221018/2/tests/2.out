0.8414709848078965
--<<UraUra>>--
все будет хорошо!! 3, 5
